### DISCUSSION TITLE
Unable to get Debug or detailed logging with keda operator logs.

### DISCUSSION THREAD

**Discussion by KasyapG** [kedacore/keda#5447] [General]
Hi, 

We are trying get detailed logging on information like

- When did the scaling start.
- Pod information
- Any specific error details.

However, Keda operator logs just show minimal info logs, we tried various ways mentioned in the blog, https://github.com/kedacore/keda/issues/3655 however all we could see is only info logs.

Could anyone please suggest the correct way or any alternative to get custom logging / debug logging on Keda operator.

Regards,
Kasyap G

**Comment by zroubalik** [comment]
Could you please point me to the specific log messages that you would like to see?
